app/models/ability.rb

```diff
@@ -45,14 +45,20 @@ def not_signed_in
     # can view Commercials of confirmed Events
     can :show, Commercial, commercialable_type: 'Event', commercialable_id: Event.where(state: 'confirmed').pluck(:id)
     can [:show, :create], User
-    unless ENV['OSEM_ICHAIN_ENABLED'] == 'true'
+    if ENV['OSEM_ICHAIN_ENABLED'] == 'true'
+      can [:new, :create], Registration do |registration|
```

Comment:
That's not accurate. This is authorization rules for a not_signed_in user. 
If ICHAIN is enabled and the user is **not** logged in, then the user **cannot** access new and create actions for Registration.